src/cbottle/inference.py

```diff
@@ -103,13 +104,27 @@ def from_pretrained(
         sigma_thresholds: tuple[float, ...] = (),
         **kwargs,
     ) -> "CBottle3d":
-        net = MixtureOfExpertsDenoiser.from_pretrained(path, sigma_thresholds)
+        allow_second_order_derivatives = kwargs.pop(
```

Comment:
It's fewer LOC and more explicit to add this to the argument list instead of manipulating kwargs.
```
        sigma_thresholds: tuple[float, ...] = (),
        allow_second_order_derivative: bool = False,
        **kwargs,
    ) -> "CBottle3d":
        net = MixtureOfExpertsDenoiser.from_pretrained(path, sigma_thresholds)
```